design/media_types.go

```diff
@@ -62,9 +62,10 @@ var meta = a.Type("workItemListResponseMeta", func() {
 // position represents the ID of the workitem above which the to-be-reordered workitem(s) should be placed
 var position = a.Type("workItemReorderPosition", func() {
 	a.Description("Position represents the ID of the workitem above which the to-be-reordered workitem(s) should be placed")
-	a.Attribute("above", d.String, "ID of the workitem above which the to-be-reordered workitem(s) should be placed")
+	a.Attribute("id", d.String, "ID of the workitem above which the to-be-reordered workitem(s) should be placed")
+	a.Attribute("direction", d.String, "position of workitem")
```

Comment:
I think the description is a bit off, don't you think? How can a `direction` be a `position` at the same time?